srcbpatch/streamreplacer.cpp

```diff
@@ -0,0 +1,544 @@
+#include "stdafx.h"
```

Comment:
Is that a good idea to include all the headers? I don't know.

Reply:
read about precompiled headers